Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -515,15 +516,15 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         }
     }
 
-    open func load(options: StartupOptions) throws {
+    open func load(options: StartupOptions) async throws {
         loadKeyValidity(from: options)
         loadSelectedEnvironment(from: options)
         loadSelectedServer(from: options)
         loadSelectedLocation(from: options)
         loadDNSSettings(from: options)
         loadTesterEnabled(from: options)
 #if os(macOS)
-        try loadAuthToken(from: options)
+            try await loadAuthToken(from: options)
```

Comment:
The indentation is off here as this is not a conditional block, but a conditionally-compiled line.